### PR TITLE
Fixed Murciélago info.json to work with qmk configurator -> wrong assignments

### PR DESCRIPTION
**Pull request by elagil**
## Description

After merging the Murciélago board to qmk (https://github.com/qmk/qmk_firmware/pull/8920), I noticed that the key assignment in QMK configurator is completely wrong. I manually had to correct the info.json, to reflect my matrix.

Also, the readme was updated in the process.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by elagil** [comment]
After some testing I suppose that QMK configurator reads the configuration row-wise from left to right and then assigns the keys to the corresponding matrix positions. Therefore, I ordered the keys accordingly (ascending x within rows, then y for consecutive rows).

**Comment by noroadsleft** [comment]
The problem you experienced is because [QMK's KLE-to-info.json Converter](https://qmk.fm/converter/) doesn't "play nice" when the keyboard uses vertical stagger. It reads the X and Y coordinates of each key, and sorts them by their Y values, then by the X values, so the output you get looks accurate, but is entirely wrong.

File: keyboards/murcielago/info.json [hunk 1]
```diff
@@ -6,7 +6,68 @@
...
+        {"label":"4,1", "x":2.2, "y":4.5},
+        {"label":"4,2", "x":3.2, "y":4.3},
+        {"label":"4,3", "x":4.2, "y":4.4},
+        {"label":"4,4", "x":5.2, "y":4.6},
+        {"label":"4,5", "x":6.3, "y":4.5, "h":1.5},
+        {"label":"10,0", "x":9.2, "y":4.5, "h":1.5},
+        {"label":"10,1", "x":10.3, "y":4.6},
+        {"label":"10,2", "x":11.3, "y":4.4},
+        {"label":"10,3", "x":12.3, "y":4.3},
+        {"label":"10,4", "x":13.3, "y":4.5}
```
Comment (noroadsleft) [inline keyboards/murcielago/info.json:69]: This change just takes out the dead space in your rendering. Your keys currently start at `y: 0.25` and `x: 0.5`. Also reduces the gap between the halves in the rendering.

```suggestion
        {"label":"0,0", "x":0, "y":0.425},
        {"label":"0,1", "x":1, "y":0.25},
        {"label":"0,2", "x":2, "y":0.125},
        {"label":"0,3", "x":3, "y":0.0},
        {"label":"0,4", "x":4, "y":0.125},
        {"label":"0,5", "x":5, "y":0.25},
        {"label":"6,0", "x":9, "y":0.25},
        {"label":"6,1", "x":10, "y":0.125},
        {"label":"6,2", "x":11, "y":0.0},
        {"label":"6,3", "x":12, "y":0.125},
        {"label":"6,4", "x":13, "y":0.25},
        {"label":"6,5", "x":14, "y":0.425},
        {"label":"1,0", "x":0, "y":1.425},
        {"label":"1,1", "x":1, "y":1.25},
        {"label":"1,2", "x":2, "y":1.125},
        {"label":"1,3", "x":3, "y":1.0},
        {"label":"1,4", "x":4, "y":1.125},
        {"label":"1,5", "x":5, "y":1.25},
        {"label":"7,0", "x":9, "y":1.25},
        {"label":"7,1", "x":10, "y":1.125},
        {"label":"7,2", "x":11, "y":1.0},
        {"label":"7,3", "x":12, "y":1.125},
        {"label":"7,4", "x":13, "y":1.25},
        {"label":"7,5", "x":14, "y":1.425},
        {"label":"2,0", "x":0, "y":2.425},
        {"label":"2,1", "x":1, "y":2.25},
        {"label":"2,2", "x":2, "y":2.125},
        {"label":"2,3", "x":3, "y":2.0},
        {"label":"2,4", "x":4, "y":2.125},
        {"label":"2,5", "x":5, "y":2.25},
        {"label":"8,0", "x":9, "y":2.25},
        {"label":"8,1", "x":10, "y":2.125},
        {"label":"8,2", "x":11, "y":2.0},
        {"label":"8,3", "x":12, "y":2.125},
        {"label":"8,4", "x":13, "y":2.25},
        {"label":"8,5", "x":14, "y":2.425},
        {"label":"3,0", "x":0, "y":3.425},
        {"label":"3,1", "x":1, "y":3.25},
        {"label":"3,2", "x":2, "y":3.125},
        {"label":"3,3", "x":3, "y":3.0},
        {"label":"3,4", "x":4, "y":3.125},
        {"label":"3,5", "x":5, "y":3.25},
        {"label":"5,5", "x":6, "y":3.25},
        {"label":"11,0", "x":8, "y":3.25},
        {"label":"9,0", "x":9, "y":3.25},
        {"label":"9,1", "x":10, "y":3.125},
        {"label":"9,2", "x":11, "y":3.0},
        {"label":"9,3", "x":12, "y":3.125},
        {"label":"9,4", "x":13, "y":3.25},
        {"label":"9,5", "x":14, "y":3.425},
        {"label":"4,1", "x":1.7, "y":4.25},
        {"label":"4,2", "x":2.7, "y":4.05},
        {"label":"4,3", "x":3.7, "y":4.15},
        {"label":"4,4", "x":4.7, "y":4.35},
        {"label":"4,5", "x":5.8, "y":4.25, "h":1.5},
        {"label":"10,0", "x":8.2, "y":4.25, "h":1.5},
        {"label":"10,1", "x":9.3, "y":4.35},
        {"label":"10,2", "x":10.3, "y":4.15},
        {"label":"10,3", "x":11.3, "y":4.05},
        {"label":"10,4", "x":12.3, "y":4.25}
```

Comment (noroadsleft) [inline keyboards/murcielago/info.json:69]: This change just takes out the dead space in your rendering. Your keys currently start at `y: 0.25` and `x: 0.5`. Also reduces the gap between the halves in the rendering.

```suggestion
        {"label":"0,0", "x":0, "y":0.425},
        {"label":"0,1", "x":1, "y":0.25},
        {"label":"0,2", "x":2, "y":0.125},
        {"label":"0,3", "x":3, "y":0.0},
        {"label":"0,4", "x":4, "y":0.125},
        {"label":"0,5", "x":5, "y":0.25},
        {"label":"6,0", "x":9, "y":0.25},
        {"label":"6,1", "x":10, "y":0.125},
        {"label":"6,2", "x":11, "y":0.0},
        {"label":"6,3", "x":12, "y":0.125},
        {"label":"6,4", "x":13, "y":0.25},
        {"label":"6,5", "x":14, "y":0.425},
        {"label":"1,0", "x":0, "y":1.425},
        {"label":"1,1", "x":1, "y":1.25},
        {"label":"1,2", "x":2, "y":1.125},
        {"label":"1,3", "x":3, "y":1.0},
        {"label":"1,4", "x":4, "y":1.125},
        {"label":"1,5", "x":5, "y":1.25},
        {"label":"7,0", "x":9, "y":1.25},
        {"label":"7,1", "x":10, "y":1.125},
        {"label":"7,2", "x":11, "y":1.0},
        {"label":"7,3", "x":12, "y":1.125},
        {"label":"7,4", "x":13, "y":1.25},
        {"label":"7,5", "x":14, "y":1.425},
        {"label":"2,0", "x":0, "y":2.425},
        {"label":"2,1", "x":1, "y":2.25},
        {"label":"2,2", "x":2, "y":2.125},
        {"label":"2,3", "x":3, "y":2.0},
        {"label":"2,4", "x":4, "y":2.125},
        {"label":"2,5", "x":5, "y":2.25},
        {"label":"8,0", "x":9, "y":2.25},
        {"label":"8,1", "x":10, "y":2.125},
        {"label":"8,2", "x":11, "y":2.0},
        {"label":"8,3", "x":12, "y":2.125},
        {"label":"8,4", "x":13, "y":2.25},
        {"label":"8,5", "x":14, "y":2.425},
        {"label":"3,0", "x":0, "y":3.425},
        {"label":"3,1", "x":1, "y":3.25},
        {"label":"3,2", "x":2, "y":3.125},
        {"label":"3,3", "x":3, "y":3.0},
        {"label":"3,4", "x":4, "y":3.125},
        {"label":"3,5", "x":5, "y":3.25},
        {"label":"5,5", "x":6, "y":3.25},
        {"label":"11,0", "x":8, "y":3.25},
        {"label":"9,0", "x":9, "y":3.25},
        {"label":"9,1", "x":10, "y":3.125},
        {"label":"9,2", "x":11, "y":3.0},
        {"label":"9,3", "x":12, "y":3.125},
        {"label":"9,4", "x":13, "y":3.25},
        {"label":"9,5", "x":14, "y":3.425},
        {"label":"4,1", "x":1.7, "y":4.25},
        {"label":"4,2", "x":2.7, "y":4.05},
        {"label":"4,3", "x":3.7, "y":4.15},
        {"label":"4,4", "x":4.7, "y":4.35},
        {"label":"4,5", "x":5.8, "y":4.25, "h":1.5},
        {"label":"10,0", "x":8.2, "y":4.25, "h":1.5},
        {"label":"10,1", "x":9.3, "y":4.35},
        {"label":"10,2", "x":10.3, "y":4.15},
        {"label":"10,3", "x":11.3, "y":4.05},
        {"label":"10,4", "x":12.3, "y":4.25}
```

**Comment by elagil** [comment]
Ok, that makes sense. Therefore, I hope that I fixed it with this pull request. I ordered the keys manually in the file, based on their x and y positions.

**Comment by elagil** [comment]
Ok, all done!

**Comment by noroadsleft** [comment]
Thanks!

**Comment by elagil** [comment]
Thanks for merging! Just tested, and it now works correctly.

**Pull request merged**